### PR TITLE
fix(ivy): support env where requestAnimationFrame is not available

### PR DESCRIPTION
**Pull request by marclaval**
I faced this issue when running the router tests with Ivy.

### CODE REVIEW

**Review by mhevery — APPROVED** [review]
Approved with changes 99d2764dcf Please verify that you are OK with it.

File: packages/router/test/integration.spec.ts [hunk 1]
```diff
@@ -389,7 +389,7 @@ describe('Integration', () => {
        expect(fixture.nativeElement).toHaveText('team 33 [ , right:  ]');
      })));
 
-  it('should work when an outlet is in an ngIf',
+  xit('should work when an outlet is in an ngIf',
```
Comment (alfaproject) [inline packages/router/test/integration.spec.ts]: This doesn't look related. Was it on purpose?

Comment (marclaval) [reply packages/router/test/integration.spec.ts]: Nope, thanks for spotting it.

**Comment by kara** [comment]
@marclaval Can you rebase this?

**Comment by marclaval** [comment]
@kara rebase done, PTAL

**Comment by kara** [comment]
[presubmit](http://test/OCL:220123056:BASE:220123224:1541441137371:b1dbaa72)

**Comment by kara** [comment]
@marclaval @mhevery Presubmit failed because Closure doesn't like the usage of `this` in util.ts

**Comment by marclaval** [comment]
@kara @mhevery I've updated the PR to use `global` from the main util.ts, I guess it should be fine now.
